tests/Application/config/packages/doctrine.yaml

```diff
@@ -12,3 +12,6 @@ doctrine:
         charset: UTF8
 
         url: '%env(resolve:DATABASE_URL)%'
+    orm:
+        resolve_target_entities:
+            Sylius\Component\Core\Model\OrderInterface: Sylius\Component\Core\Model\Order
```

Comment:
Why did you add it here?

Reply:
For some reason, it was not working without... I was doing a quick PoC so I've added it and decided to resolve it later. @AdamKasp now is "later" 😄